Packages/manifest.json

```diff
@@ -8,6 +8,7 @@
     "com.unity.ide.visualstudio": "2.0.22",
     "com.unity.ide.vscode": "1.2.5",
     "com.unity.inputsystem": "1.5.1",
+    "com.unity.recorder": "3.0.4",
```

Comment:
are these changes necessary for the powerShot to work?
https://discord.com/channels/388747322130956288/1262597209954844712/1266879039990403112

Reply:
I tested the changes without the packages and it still seems to work so recorder shouldn't be needed